packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -406,7 +407,29 @@ module.exports = {
       fileName: 'asset-manifest.json',
       publicPath: publicPath,
     }),
-  ],
+    // TypeScript type checking
+    fs.existsSync(paths.appTSConfig) &&
+      (() => {
+        let ForkTsCheckerWebpackPlugin;
+        try {
+          ForkTsCheckerWebpackPlugin = require(`${
+            paths.appNodeModules
```

Comment:
We can't require this directly from the user's `appNodeModules` (think workspace hoisting) -- we should probably just use a normal require or [`resolve-from`](https://github.com/sindresorhus/resolve-from) and feed it this path.

Reply:
Couldn't make it work with normal requires; we can try this resolve-from / import-from

Reply:
@Timer `resolve-from` fixed the issue 🖤

Reply:
@Timer True. Updated.